package.json

```diff
@@ -40,6 +40,8 @@
     "async": "2.6.1",
     "babel-core": "6.26.3",
     "babel-loader": "7.1.5",
+    "babel-plugin-transform-es2015-spread": "6.22.0",
+    "babel-plugin-transform-object-rest-spread": "6.26.0",
```

Comment:
I think these plugins are not necessary. babel presets env should already handle this.